packages/webex/package.json

```diff
@@ -46,6 +46,7 @@
     "@webex/common": "workspace:^",
     "@webex/internal-plugin-calendar": "workspace:^",
     "@webex/internal-plugin-device": "workspace:^",
+    "@webex/internal-plugin-llm": "workspace:^",
```

Comment:
these changes are a bit suss, they should already be in master. maybe try to merge `beta` into your branch

Reply:
noup, it's still not on beta

Reply:
i meant they should already be on `beta`